src/Mod/Part/BasicShapes/SkipGui.py

```diff
@@ -0,0 +1,271 @@
+#! python
+# -*- coding: utf-8 -*-
+# SkipGui.py
+# 2022, by Mark Ganson <TheMarkster>
+# LGPL 2.1 or later
+
+# this file is called from c++ in Mod/Part/Gui/Command.cpp
+# from various functions when these commands are activated:
+#   Part::Loft
+#   Part::Sweep
+#   Part::Extrude
+#   Part::Revolve
+# Then if the user has preselected certain objects and subobjects
+# the Gui can be skipped and the document objects directly created
+
+# If False is returned, then the Gui dialog should be shown
+# Else if True is returned, then the dialog can be skipped
+
+import FreeCAD as App
+import FreeCADGui as Gui
+
+def getDocStr(document):
+    return f"App.getDocument('{document.Name}')"
+
+def getActStr(document):
+    return f"{getDocStr(document)}.ActiveObject"
+
+def getObjStr(document, object):
+    """App.getDocument(document).getObject(object)"""
+    return f"{getDocStr(document)}.getObject('{object.Name}')"
+
+def setHiddenStr(document, objs):
+    """App.getDocument(document).getObject('Sketch').ViewObject.Visibility = False"""
```

Comment:
I'd actually expect something more human readable, as "Returns a command line that will hide passed objects"

Same for other docstrings.

Reply:
Oops.  Yeah, I forgot to update CMakeLists.txt when I made the new branch.  I'm curious why my tests worked without .py file now.  I guess it was using the old one.  I build always no matter the branch into the same build folder called masterbin.

Good idea on the doc strings.